### PR TITLE
Generalize README for OS and languages

### PR DESCRIPTION
**Pull request by Meadosc**
Generalize the README so OS specific commands are not used and python language version is accounted for in virtual environment discussion

### CODE REVIEW

**Comment by Meadosc** [comment]
My only misgiving is the realpython link for virtual environments. It is a better run down than we can provide, but I question the professionalism.

**Comment by Meadosc** [comment]
@VictorRodriguez

File: README.md [hunk 1]
```diff
@@ -13,54 +13,40 @@ This tool is just a glorified dockerfiles and singularity recipie sticher with a
...
-```
-### Install the project requirements on virtual environment.
-Now that virtualenv is activated you need to install the needed packages.
+
+- Install requirements
 
 ```
- $ cd <stacker>
- $ pip install -r requirements.txt
+pip install requirements.txt
```
Comment (DnPlas) [inline README.md]: Please add the -r flag

File: README.md [hunk 2]
```diff
@@ -13,54 +13,40 @@ This tool is just a glorified dockerfiles and singularity recipie sticher with a
 
 ### Project requirements
 
+It is common to use pip and a virtual environment for python development. Consider using these tools while following this README. If you are using python 3 'venv' will already be installed, and all testing is done in python 3.
```
Comment (DnPlas) [inline README.md]: I would add this paragraph under a different section, users have the option to use venv, but that's not a requirement. Maybe add this on another one called for instance "Running on venv". Also, I don't quite like the realpython link, don't think it is necessary here.

**Comment by DnPlas** [comment]
Some comments, thanks @Meadosc

File: README.md (hunk 2 above)
Comment (Meadosc) [reply README.md]: I'm going to remove the virtual environment references. We can leave that to the user

**Comment by Meadosc** [comment]
Decided to nuke the virtual environment and pip references. We can leave that for users to decide.

**Comment by DnPlas** [comment]
+1

**Pull request merged**